tests/test_transform.py

```diff
@@ -180,10 +180,94 @@ def checkTransform(self, fromName, toName):
+        transformClassName = "Transform{}To{}".format(fromName, toName)
+        transformClass = getattr(afwGeom, transformClassName)
+        baseMsg = "transformClass={}".format(transformClass.__name__)
+        for nIn in self.goodNaxes[fromName]:
+            for nOut in self.goodNaxes[toName]:
+                # Transform with both forward and inverse transforms
+                self.checkGetInverseWithMap(
+                    transformClass(makePolyMap(nIn, nOut)),
+                    nIn,
+                    "{}, Map={}, nIn={}, nOut={}".format(
+                        baseMsg, "Invertible", nIn, nOut))
+                # TODO: add more test cases once DM-9826 resolved
+
+    def checkGetInverseWithMap(self, transform, nIn, msg):
+        """Test Transform<fromName>To<toName>.getInverse with a specific mapping.
+
+        Parameters
+        ----------
+        transform : Transform
+            the transform to test
+        nIn, nOut : integer
+            the number of dimensions in `transform`'s input and output
+        msg : string
+            a suffix for error messages, distinguishing this test from others
+        """
+        inverse = transform.getInverse()
+        # no equality test for FrameSets... do it the hard way
+        frameSet = transform.getFrameSet()
+        invFrameSet = inverse.getFrameSet()
+
+        # properties
+        self.assertEqual(transform.getFromEndpoint(),
+                         inverse.getToEndpoint(), msg=msg)
+        self.assertEqual(transform.getToEndpoint(),
+                         inverse.getFromEndpoint(), msg=msg)
+        # TODO: test hasTranForward and hasTranInverse once DM-9826 resolved
+
+        # transformations of one point
+        fromEndpoint = transform.getFromEndpoint()
+        toEndpoint = transform.getToEndpoint()
+        rawInPoint = makeRawPointData(nIn)
+        inPoint = fromEndpoint.pointFromData(rawInPoint)
+        outPoint = transform.tranForward(inPoint)
+        rawOutPoint = toEndpoint.dataFromPoint(outPoint)
+        self.assertEqual(transform.tranForward(inPoint),
+                         inverse.tranInverse(inPoint), msg=msg)
+        self.assertEqual(transform.tranInverse(outPoint),
+                         inverse.tranForward(outPoint), msg=msg)
+        self.assertEqual(frameSet.tranForward(rawInPoint),
+                         invFrameSet.tranInverse(rawInPoint), msg=msg)
+        self.assertEqual(frameSet.tranInverse(rawOutPoint),
+                         invFrameSet.tranForward(rawOutPoint), msg=msg)
+
+        # transformations of arrays of points
+        nPoints = 7  # arbitrary
+        rawInArray = makeRawArrayData(nPoints, nIn)
+        inArray = fromEndpoint.arrayFromData(rawInArray)
+        outArray = transform.tranForward(inArray)
+        rawOutArray = toEndpoint.dataFromArray(outArray)
+        # Assertions must work with both lists and numpy arrays
+        # TODO replace with assertFloatsAlmostEqual once DM-9707 is fixed
```

Comment:
My suggestion would be to avoid embedding TODOs like this in the code — we have literally hundreds of them scattered across the codebase, and nobody has time to regularly audit old code to see what actually needs to be done. Perhaps you could file a ticket on JIRA to do this work, and mark it as triggered by DM-9707? (Obviously there are _thousands_ of tickets in JIRA which need keeping track of, but at least that means we only need to track a single source.)